SurveyApp/Modules/Home/HomeInteractor.swift

```diff
@@ -6,20 +6,68 @@
 //  
 //
 
+import Foundation
+
 // sourcery: AutoMockable
 protocol HomeInteractorInput: AnyObject {
+    
+    func fetchSurveys(pageNumber: Int, pageSize: Int)
 }
 
 // sourcery: AutoMockable
 protocol HomeInteractorOutput: AnyObject {
+    
+    func didFetchSurveys(surveys: [Survey])
+    func didFailToFetchSurveys()
 }
 
 final class HomeInteractor {
 
     weak var output: HomeInteractorOutput?
+    
+    @UserDefault(key: .cachedSurveyList, defaultValue: [Survey]())
+    var cachedSurveyList: [Survey]
```

Comment:
please create enum `UserStorage` for this and use it like UserStorage. cachedSurveyList